seL4-platforms/platforms.yml

```diff
@@ -260,7 +283,7 @@ platforms:
 mcs_unsupported_platforms:
 # tx1 - scheduler accuracy fails, problem with clk rate in kernel or timestamp
 # at user level?
-- TX1
+# - TX1
```

Comment:
no idea what this commenting out implies

Reply:
It's hidden in the commit message :-). The not-supported-for-mcs list is from Bamboo, and it mostly checks the list, but it has a special case for some unrelated TX1 stuff that happens to add MCS tests back into the queue. So it's not actually unsupported. To get the test set exactly equal with what is running on Bamboo, I removed it here.